examples/one_degree_simulation.jl

```diff
@@ -0,0 +1,226 @@
+##### Grid and Bathymetry
+#####
+
+Nx = 360
+Ny = 180
+Nz = 100
+
+z_faces = exponential_z_faces(; Nz, depth=5000, h=34)
+
+underlying_grid = TripolarGrid(arch;
+                               size = (Nx, Ny, Nz),
+                               z = z_faces,
+                               halo = (5, 5, 4),
+                               first_pole_longitude = 70,
+                               north_poles_latitude = 55)
+
+bottom_height = regrid_bathymetry(underlying_grid;
+                                  minimum_depth = 10,
+                                  interpolation_passes = 75,
+                                  major_basins = 2)
+
+# Open Gibraltar strait 
+# TODO: find a better way to do this
+tampered_bottom_height = deepcopy(bottom_height)
+view(tampered_bottom_height, 102:103, 124, 1) .= -400
+
+grid = ImmersedBoundaryGrid(underlying_grid, GridFittedBottom(tampered_bottom_height); active_cells_map=true)
+
+#####
+##### Closures
+#####
+
+# We don't need any diffusion: κ_symmetric = 0!
+eddy_closure = Oceananigans.TurbulenceClosures.IsopycnalSkewSymmetricDiffusivity(κ_skew=1000) 
+vertical_mixing = ClimaOcean.OceanSimulations.default_ocean_closure()
+
+closure = (eddy_closure, vertical_mixing)
+
+#####
+##### Restoring
+#####
+
+restoring_rate  = 1 / 10days
+z_below_surface = z_faces[end-1]
+
+mask = LinearlyTaperedPolarMask(southern=(-80, -70), northern=(70, 90), z=(z_below_surface, 0))
+
+dates = DateTimeProlepticGregorian(1993, 1, 1) : Month(1) : DateTimeProlepticGregorian(1993, 11, 1)
+temperature = ECCOMetadata(:temperature; dates, version=ECCO4Monthly(), dir="./")
+salinity    = ECCOMetadata(:salinity;    dates, version=ECCO4Monthly(), dir="./")
+
+FT = ECCORestoring(temperature, grid; mask, rate=restoring_rate)
+FS = ECCORestoring(salinity,    grid; mask, rate=restoring_rate)
+forcing = (T=FT, S=FS)
+
+#####
+##### Ocean simulation
+##### 
+
+momentum_advection = WENOVectorInvariant(vorticity_order=3)
+tracer_advection   = Centered()
+
+using Oceananigans.TurbulenceClosures: IsopycnalSkewSymmetricDiffusivity,
+                                       ExplicitTimeDiscretization,
+                                       DiffusiveFormulation
+
+using Oceananigans.TurbulenceClosures.TKEBasedVerticalDiffusivities: CATKEVerticalDiffusivity
+
+numerical_closure = HorizontalScalarDiffusivity(ν=5e3)
+eddy_closure = IsopycnalSkewSymmetricDiffusivity(κ_skew=1e3, κ_symmetric=1e3, skew_flux_formulation=DiffusiveFormulation())
+vertical_mixing = CATKEVerticalDiffusivity()
+
+closure = (eddy_closure, numerical_closure, vertical_mixing)
+
+# Spacings still don't work correctly?
+free_surface = SplitExplicitFreeSurface(grid; substeps=30)
+
+# Should we add a side drag since this is at a coarser resolution?
+ocean = ocean_simulation(grid;
+                         momentum_advection,
+                         tracer_advection,
+                         closure,
+                         forcing,
+                         free_surface)
+
```

Comment:
```suggestion
set!(ocean.model, T=ECCOMetadata(:temperature; dates=first(dates)),
                  S=ECCOMetadata(:salinity; dates=first(dates)))
```

Reply:
This errors for me:
```
[ Info: Time: 101.875 days, n: 7290, Δt: 30 minutes, max|u|: (NaN, NaN, NaN) m s⁻¹, extrema(T): (NaN, NaN) ᵒC, wall time: 10.618 seconds 
[ Info: time = 8.82e6, iteration = 7300: NaN found in field u_ocean. Stopping simulation.
```

Reply:
With those values set, I'm up to 262 days ~15k timesteps.

Reply:
good catch!